spec/models/room_spec.rb

```diff
@@ -6,6 +6,9 @@
   it { is_expected.to have_many(:gizmos).inverse_of(:room).dependent(:destroy) }
   it { is_expected.to belong_to(:space).inverse_of(:rooms) }
 
+  it { is_expected.to belongs_to(:hero_image) }
+  it { is_expected.to belongs_to(:feed_image) }
```

Comment:
I think this wants to be singular (`belong_to` because rspec/rubyists *LOVE* grammar!)